lib/rucio/daemons/judge/cleaner.py

```diff
@@ -151,9 +151,10 @@ def run(once=False, threads=1):
     """
     client_time, db_time = datetime.utcnow(), get_db_time()
     max_offset = timedelta(hours=1, seconds=10)
-    if db_time - client_time > max_offset or client_time - db_time > max_offset:
-        logging.critical('Offset between client and db time too big. Stopping Cleaner')
-        return
+    if type(db_time) is datetime:
```

Comment:
what if it's not? safe to continue?

Reply:
It's the best you can do. Certain version of postgres only return a date, thus you cannot find out the actual db time that easily. (At least not via the sqlalchemy method). But this check is really an overly precautious safeguard. Not really needed.